src/store/configureStore.js

```diff
@@ -114,9 +120,39 @@ const middlewares = applyMiddleware(
 
 let store = null
 
+// As long as the mobile client is dependent on the web client, we need to sync
+// the client store from web -> mobile
+const clientStoreKeys = Object.keys(clientStoreReducers)
```

Comment:
not that this is super huge, but can we put this in a Set? would make the lookup on each subscription O(1)